neurom/features/morphology.py

```diff
@@ -220,7 +293,7 @@ def neurite_volume_density(morph, neurite_type=NeuriteType.all):
 
 
 @feature(shape=(...,))
-def sholl_crossings(morph, center=None, radii=None, neurite_type=NeuriteType.all):
+def sholl_crossings(morph, neurite_type=NeuriteType.all, center=None, radii=None):
```

Comment:
please also move `neurite_type` in docstring

Reply:
ups, sorry :)
Done